### DISCUSSION TITLE
Saving a To Do or new User Gives a "This Page Isn't Working" error

### DISCUSSION THREAD

**Discussion by tedcreynolds** [Leantime/leantime#695] [General]
I installed LeanTime using the docker-compose method on both Fedora and Ubuntu server.
Every time I save a "To do" whether a quick to do or regular to do I get a "This page isn't working" page.  The to do is actually saved which you can see by clicking the browser's back button.  Also get an "An error occurred" page after saving any new user.  There is no error shown in the page just a red outlined box with the words "An Error Occurred."  Just like the to do problem if you click the browser's back button this page goes away and the user is actually saved.

**Comment by marcelfolaron** [comment]
Please try to set the email variable in your config file. We are working on
a fix to handle this issue more gracefully
